llama_hub/wikipedia/README.md

```diff
@@ -15,4 +17,9 @@ loader = WikipediaReader()
 documents = loader.load_data(pages=['Berlin', 'Rome', 'Tokyo', 'Canberra', 'Santiago'])
 ```
 
-This loader is designed to be used as a way to load data into [LlamaIndex](https://github.com/jerryjliu/gpt_index/tree/main/gpt_index) and/or subsequently used as a Tool in a [LangChain](https://github.com/hwchase17/langchain) Agent. See [here](https://github.com/emptycrown/llama-hub/tree/main) for examples.
+This loader is designed for loading data into [LlamaIndex](https://github.com/jerryjliu/gpt_index/tree/main/gpt_index)
+and/or subsequently used as a Tool in a [LangChain](https://github.com/langchain-ai/langchain) Agent.
+See [this LlamaIndex tutorial][1] for examples.
```

Comment:
This readme is only for Reader (load documents). Doesn't give the methods necessary for tooling, I think the deal would be to update the Tool readme as well, what do you think?

Wikipedia Tool: https://github.com/emptycrown/llama-hub/tree/main/llama_hub/tools/wikipedia

Reply:
Oh haha so I edited the wrong README 😆 will fix this

Reply:
Alright @EmanuelCampos I fixed it up, lemme know what you think!